test/Compiler.test.js

```diff
@@ -258,6 +259,74 @@ describe("Compiler", () => {
 			done();
 		});
 	});
+
+	it("should bubble up errors when wrapped in a promise and bail is true", done => {
+		const asyncCompile = options => {
+			return new Promise((resolve, reject) => {
+				const c = webpack(options);
```

Comment:
Declarative names like `compiler` is favorable from me

Reply:
👍

Reply:
Initially I was using async/await but was getting a `no-unsupported-features` error because of the Node version being used.